model/patch_lifecycle.go

```diff
@@ -438,9 +438,55 @@ func FinalizePatch(ctx context.Context, p *patch.Patch, requester string, github
 		return nil, errors.WithStack(err)
 	}
 
+	if p.IsParent() {
+		//finalize all child patches that have a parentStatus = ""
+		for _, childPatch := range p.Triggers.ChildPatches {
+			err = finalizeChildPatch(ctx, childPatch, requester, githubOauthToken)
+			if err != nil {
+				return nil, errors.Wrap(err, "unable to finalize child patch")
+			}
+			return nil, err
```

Comment:
why do we return here in the not error case?

Reply:
I think that's a copy paste error. I'll remove it. thanks!